libr/util/str.c

```diff
@@ -3573,3 +3574,23 @@ R_API char *r_str_scale(const char *s, int w, int h) {
 	free (str);
 	return r_str_list_join (out, "\n");
 }
+
+
+
+R_API int r_charset_encode_str(ut8 *asciistr, ut8 *in, size_t len_input, charset custom_charset, size_t len_charset) {
+	int i;
+	int j;
+	for(i = 0; i < len_input; i ++)
```

Comment:
*  open braces at right
* use size_t for loop variables
* missing space before (
* remove space before ++

Reply:
@radare the braces are always opened in the declarations of this file

> open braces at right

Reply:
i dont understand your response here. This function have syntax indentation issues, please solve them